docs/use_cases/met_tool_wrapper/PointStat/PointStat_python_embedding.py

```diff
@@ -0,0 +1,136 @@
+
+met_tool_wrapper/PointStat/PointStat_python_embedding.conf
+
+"""
+##############################################################################
+# Scientific Objective
+# --------------------
+#
+# Read forecast data using a Python embedding script. Compare with point
+# observation data.
+
+##############################################################################
+# Datasets
+# --------
+#
+# | **Forecast:** NRL binary data (v-wind component)
+# | **Observation:** prepBUFR data that has been converted to NetCDF format via PB2NC
+#
+# | **Location:** All of the input data required for this use case can be found in the met_test sample data tarball. Click here to the METplus releases page and download sample data for the appropriate release: https://github.com/dtcenter/METplus/releases
+# | This tarball should be unpacked into the directory that you will set the value of INPUT_BASE. See 'Running METplus' section for more information.
+#
+# | **Data Source:** Unknown
+
+##############################################################################
+# METplus Components
+# ------------------
+#
+# This use case utilizes the METplus PointStat wrapper to search for
+# files that are valid at a given run time and generate a command to run
+# the MET tool point_stat if all required files are found. The forecast data
+# is read into PointStat using a Python embedding script
+
+##############################################################################
+# METplus Workflow
+# ----------------
+#
+# PointStat is the only tool called in this example. It processes the following
+# run times:
+#
+# | **Valid:** 2020-09-06 6Z
+
+##############################################################################
+# METplus Configuration
+# ---------------------
+#
+# METplus first loads all of the configuration files found in parm/metplus_config,
+# then it loads any configuration files passed to METplus via the command line
+# with the -c option, i.e. -c parm/use_cases/met_tool_wrapper/PointStat/PointStat_python_embedding.conf
+#
+# .. highlight:: bash
+# .. literalinclude:: ../../../../parm/use_cases/met_tool_wrapper/PointStat/PointStat_python_embedding.conf
+
+##############################################################################
+# MET Configuration
+# ---------------------
+#
+# METplus sets environment variables based on the values in the METplus configuration file.
+# These variables are referenced in the MET configuration file. **YOU SHOULD NOT SET ANY OF THESE ENVIRONMENT VARIABLES YOURSELF! THEY WILL BE OVERWRITTEN BY METPLUS WHEN IT CALLS THE MET TOOLS!** If there is a setting in the MET configuration file that is not controlled by an environment variable, you can add additional environment variables to be set only within the METplus environment using the [user_env_vars] section of the METplus configuration files. See the 'User Defined Config' section on the 'System Configuration' page of the METplus User's Guide for more information.
```

Comment:
This appears to be one very long line with no auto-wrapping, not sure if we should add in some line breaks.

Reply:
I copied most of the content of this file from another use case doc file, including this line. I agree that this should be split into multiple lines, but this will need to be done for many of these files. This is related to issue #644 although I don't think the use case docs show up in the "line too long" errors since they are generated from Sphinx Gallery. I can add a note to that issue to review and clean up these files as well.